index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    
+    <header class="toolbar">
+     
+       <nav ><a class="logo" href="#"><img src="Images\Logo Placeholder.png" alt="logo" /></a>
+        <img class="icons" src="Images\Enabled.png" alt="threeline" /></nav> 
```

Comment:
- [ ] Good job trying :+1:  to emplement the requirements suggested by the previous, Kindly follow the structure below to satisfy the conditions he raised:
```
<header>
<a href="#">Logo</a>
<nav> Humburger </a>
</header>
```